analysis_options.yaml

```diff
@@ -0,0 +1,159 @@
+# Specify analysis options.
+#
+# Until there are meta linter rules, each desired lint must be explicitly enabled.
+# See: https://github.com/dart-lang/linter/issues/288
+#
+# For a list of lints, see: http://dart-lang.github.io/linter/lints/
+# See the configuration guide for more
+# https://github.com/dart-lang/sdk/tree/master/pkg/analyzer#configuring-the-analyzer
+#
+# There are four similar analysis options files in the flutter repos:
+#   - analysis_options.yaml (this file)
+#   - analysis_options_repo.yaml
+#   - packages/flutter/lib/analysis_options_user.yaml
+#   - https://github.com/flutter/plugins/blob/master/analysis_options.yaml
```

Comment:
Five now I guess?

Reply:
oh hey there's a way to import one from the other now. Maybe we can do that here! @devoncarew or @pq may know more about the syntax for doing this.

Reply:
Yes, that would be great.  Can I do it this way for now, and just switch it in another PR once I know more about how to do it?

Reply:
Sure.

Reply:
Yeah, you can reference a parent analysis options file (including by a package: url if the package is available in your deps), get all the settings from that file, and choose to override individual settings.

There's an example of that here: https://github.com/flutter/flutter/pull/16979/files#diff-60bf9684f6753f1cf7159920ecc36a89

Thinking about it now, I'd need tp confirm the `package:` reference.